data/cvr/samples/jetsons/jetsons_main_edf.xml

```diff
@@ -99,30 +117,98 @@
                 <Text Language="en">Harlan Ellis</Text>
             </BallotName>
         </Candidate>
-        <Candidate ObjectId="candidate-rudy-indexer">
+        <Candidate ObjectId="candidate-rudi-indexer">
             <BallotName>
-                <Text Language="en">Rudy Indexer</Text>
+                <Text Language="en">Rudi Indexer</Text>
             </BallotName>
         </Candidate>
-
         <Contest xsi:type="CandidateContest" ObjectId="contest-mayor-orbit-city">
+            
+            <ContestSelection xsi:type="CandidateSelection"
+                ObjectId="contest-mayor-selection-cosmo-spacely">
+                <SequenceOrder>1</SequenceOrder>
+                <CandidateIds>candidate-cosmo-spacely</CandidateIds>
+                <EndorsementPartyIds>party-lepton</EndorsementPartyIds>
+            </ContestSelection>
+
+            <ContestSelection xsi:type="CandidateSelection"
+                ObjectId="contest-mayor-selection-spencer-cogswell">
+                <SequenceOrder>2</SequenceOrder>
+                <CandidateIds>candidate-spencer-cogswell</CandidateIds>
+                <EndorsementPartyIds>party-hadron</EndorsementPartyIds>
+            </ContestSelection>
+
+            <ContestSelection xsi:type="CandidateSelection"
+                ObjectId="contest-mayor-selection-write-in">
+                <SequenceOrder>3</SequenceOrder>
+                <IsWriteIn>true</IsWriteIn>
+            </ContestSelection>
+
             <ElectionDistrictId>gadget-county</ElectionDistrictId>
+            <ExternalIdentifier>
+                <Type>local-level</Type>
+                <Value>gc-mayor-orbit-city</Value>
+            </ExternalIdentifier>
             <Name>Mayor of Orbit City</Name>
             <VoteVariation>plurality</VoteVariation>
             <VotesAllowed>1</VotesAllowed>
         </Contest>
+
         <Contest xsi:type="CandidateContest" ObjectId="contest-control-board-spaceport">
+            <ContestSelection xsi:type="CandidateSelection"
+                ObjectId="contest-control-board-spaceport-selection-jane-jetson">
```

Comment:
See note "ContestSelectionId Separators" at the top of the review.

Reply:
I've changed all of these to contest-X--selction-Y.